pkgs/tools/security/ghidra/extensions/ret-sync/default.nix

```
@@ -14,7 +15,14 @@ buildGhidraExtension {
     rev = "0617c75746ddde7fe2bdbbf880175af8ad27553e";
     hash = "sha256-+G5ccdHnFL0sHpueuIYwLRU9FhzN658CYqQCHCBwxV4=";
   };
-
+  patches = [
+    # This patch is needed to get the extension compiling with Ghidra 11.2.
+    # Once it's fixed upstream, the src can be updated and this can be removed.
+    (fetchpatch {
+      url = "https://github.com/bootleg/ret-sync/pull/126.patch";
```

please use a reproducible url

```suggestion
      # https://github.com/bootleg/ret-sync/pull/126
      name = "something.patch";
      url = "https://github.com/bootleg/ret-sync/commit/d81d953c24b4369b499e90ba64c1c9f78513a008.patch";
```